.github/workflows/bls-signatures.yml

```diff
@@ -22,3 +22,29 @@ jobs:
       run: |
         ./build/src/runtest
 
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
+
+    - name: Setup Go environment
+      run: |
+        mkdir -p go/src/github.com/dashpay
+        ln -s $(pwd) $(pwd)/go/src/github.com/dashpay/bls-signatures
+        go get -v -t -d ./...
+      env:
+        GOPATH: /home/runner/work/bls-signatures/bls-signatures/go
+
+    - name: Build go-bindings
+      run: |
+        cd go/src/github.com/dashpay/bls-signatures/go-bindings
+        go build -v .
```

Comment:
Why don't you use make targets https://github.com/dashpay/bls-signatures/blob/develop/go-bindings/Makefile